index.js

```diff
@@ -60,3 +62,46 @@ module.exports.bemhtml = function(options) {
 module.exports.bemtree = function(options) {
     return module.exports(options, 'bemtree');
 };
+
+module.exports.toHtml = function(tmpl) {
+    return geval().pipe(through.obj(function(bemjsonFile, encoding, callback) {
+        if (bemjsonFile.isNull()) {
+            return callback(null, bemjsonFile);
+        }
+        if (bemjsonFile.isStream()) {
+            return callback(new PluginError(pluginName, 'Streaming not supported'));
+        }
+
+        if (!isStream(tmpl)) {
+            return callback(new PluginError(pluginName, 'Parameter should be a Stream'));
+        }
+
+        tmpl
+            .pipe(through.obj(function(file, encoding, callback){
+                if (file.isStream()) {
+                    return callback(new PluginError(pluginName, 'Substreaming not supported'));
+                }
+                return callback(null, file);
+            }))
+            .pipe(geval())
+            .pipe(through.obj(function(file, encoding, _callback) {
+                if (file.isNull()) {
+                    return callback(null, file);
+                }
+
+                try {
+                    var bemjson = eval(bemjsonFile.contents + '');
+                    var html = file.data.apply(bemjson); // bemjsonFile.data);
+                } catch (e) {
+                    return callback(e);
```

Comment:
This tryCatch should be made like a separate function:
```js
var html = tryCatch(function() {
    var bemjson = eval(bemjsonFile.contents + ''); // bemjsonFile.data);
    var html = file.data.apply(bemjson);
});
if (html instanceof Error) {
  return callback(html);
}
```

And need to replace `eval` with `bemjsonFile.data`.